README.md

```diff
@@ -1,2 +1,62 @@
-# ably-labs-template-repo
-This is a template repository to be used for all Ably Labs demos, tools &amp; proof of concepts.
+# Ably Labs Template Repo
+
+---
+
+// This is a template repository to be used for all Ably Labs demos, tools &amp; proof of concepts. Follow these steps to so this repo is easy to use for visitors & maintainers.
+
+1. Update the description of this repo.
+2. Add [topics](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/classifying-your-repository-with-topics) to this repo to clarify the language, tech stack and use case.
+3. Update the [.gitignore](.gitignore) file with one of the [standard templates from GitHub](https://github.com/github/gitignore).
+4. Update [dependabot.yml](.github/dependabot.yml) with the  [configuration for your project](https://docs.github.com/en/code-security/supply-chain-security/keeping-your-dependencies-updated-automatically/configuration-options-for-dependency-updates).
+5. Replace `REPO_NAME` with the actual name of the repo in the [CONTRIBUTING.md](CONTRIBUTING.md) file.
+6. Update this README so it provides enough information for people to understand how it works, how to run it locally and how it can be deployed to the cloud (see [GitHub](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes)).
+7. Add a GitHub workflow to build/test/deploy your application. Use the [Ably Control API GitHub action](https://github.com/ably-labs/ably-control-api-action) to avoid creating Ably apps/API keys manually.
+
+Once you're done, remove this section from the README. Good luck! 💪
+
+---
+
+![Place eye candy header image here]()
+
+## Description
+
+// Explanation of the contents of the repository. Describe the use case.
+
+## Tech stack
+
+![A high level architecture diagram tells more than a 1000 words]()
+
+The project uses the following components:
+
+- [X](), brief explanation of the component
+- [Y](), brief explanation of the component
+- [Ably](https://ably.com/), for realtime messaging at scale.
+
+## Building & running locally
+
+### Prerequisites
+
+1. [Sign up](https://ably.com/signup) or [log in](https://ably.com/login) to ably.com, and [create a new app and copy the API key](https://faqs.ably.com/setting-up-and-managing-api-keys).
+2. Install X
+3. Install Y
+
+### Building the project
+
+1. 
+2.
+## Deploying to the cloud
+
+1.
+2.
+
+
+## Contributing
+
+We have a [contributing guide](CONTRIBUTING.md) that explains how to contribute to this repository.
+
+## More info
+
+// Link to blog, video, twitter handle.
```

Comment:
- https://twitter.com/ablyrealtime
- https://www.linkedin.com/company/ably-realtime
- https://github.com/ably/
- https://ably.com/sign-up

these are links from production

Reply:
I was thinking more one a personal level, that links to the creator of the repo and specific content they've created. But adding some of the corp ones should be ok as well.